recipes/py-dom-xpath-six/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "py-dom-xpath-six" %}
+{% set version = "0.2.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: d6c319f23ee6e71d415f3a588f6d245495c5d41cb2d2a35bd80777c4a6d2e50b
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+    - future
+  run:
+    - python >=3.6
+    - future
+
+test:
+  imports:
+    - xpath
+  requires:
+    - pip
+  commands:
+    - pip check
+
+about:
+  home: https://github.com/jackjansen/py-dom-xpath-six
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
```

Comment:
Please add a comment above the `license_file: LICENSE` line that reads:

```
# license file manually packaged. See https://github.com/jackjansen/py-dom-xpath-six/issues/3
license_file: LICENSE
```

This will be a reminder to check if the license file becomes part of the source distribution. When it does you can remove the manually packaged copy and delete the comment.

Reply:
Above comment is added in the last commit